modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Rating.es.js

```diff
@@ -90,9 +88,7 @@ export default ({
 			<span className="c-px-2">{rating || 0}</span>
 
 			<ClayButton
-				className={
```

Comment:
Are we still using classNames library because it can improve this code readibility